README.rst

```diff
@@ -98,12 +98,15 @@ Quick Example
 
     bot = commands.Bot(command_prefix='$')
 
-    @bot.command()
-    async def ping(ctx):
-        await ctx.reply('Pong!')
 
-    bot.run('token')
+    @bot.slash_command(description="Replies with pong!")
+    async def ping(interaction: nextcord.Interaction):
+        await interaction.send("Pong!", ephemeral=True)
 
+    bot.run("token")
+
+
+It is recommended that you use ``commands.Bot``, as although both ``nextcord.Client`` and ``commands.Bot`` allow you to interface with Discord, ``commands.Bot`` provides an easier way to make prefix commands with a framework. Make sure to not use both versions, though, as only one of them will actually be connected to Discord.
```

Comment:
id specify that its recommended *if you wish to add prefix commands*?

Reply:
I think it's still recommended due to Cogs, listeners, some checks?, error handling, among other things only existing with commands.Bot. It would be nice if it didn't have to all be connected, but that's how it is as of now I believe.

Reply:
Updated, please check now.

Reply:
I think it was better before. It's kind of repetitive to say twice that its recommended.

If you're going to say its recommended for prefix commands (Epic's suggestion), that should go at the beginning of the sentence (eg. "It is recommended that you use ``commands.Bot`` if you wish to add prefix commands, ..."), although I think it's not even needed at all to specify and it was fine as it was before.

Just waiting on @TAG-Epic to see what they think it should say given my comment.

Reply:
Ok, will fix something real quick as I see some wording error.